package.json

```diff
@@ -18,6 +18,9 @@
   },
   "homepage": "https://github.com/greenfox-academy/epam-jsa-black-pearl#readme",
   "dependencies": {
+    "body-parser": "^1.18.2",
```

Comment:
i don't think that you need body parser for the heartbeat endpoint since you are not receiving any JSON formatted data, you just send it.